pkg/loki/loki.go

```diff
@@ -356,6 +356,7 @@ type Loki struct {
 	tableManager             *index.TableManager
 	frontend                 Frontend
 	ruler                    *base_ruler.Ruler
+	ruleEvaluator            ruler.Evaluator
```

Comment:
Q: `ruleEvaluator` runs as a part of `ruler` component/process correct?

Reply:
Correct